src/Navigation/index.js

```diff
@@ -30,29 +35,34 @@ function Navigation() {
     };
 
     return (
-        <div className={classes.root}>
-            <AppBar position="static">
-                <Tabs
-                    variant="fullWidth"
-                    value={value}
-                    onChange={handleChange}
-                    aria-label="nav tabs example"
-                >
-                    <LinkTab label="Posts" href="/drafts" {...a11yProps(0)} />
-                    <LinkTab label="Following" href="/trash" {...a11yProps(1)} />
-                    <LinkTab label="Page Three" href="/spam" {...a11yProps(2)} />
-                </Tabs>
-            </AppBar>
-            <TabPanel value={value} index={0}>
-                <Posts />
-            </TabPanel>
-            <TabPanel value={value} index={1}>
-                <Following />
-            </TabPanel>
-            <TabPanel value={value} index={2}>
-                Page Three
-            </TabPanel>
-        </div>
+        <Router>
```

Comment:
This file is already doing too much. 

1. Rendering AppBar
2. Rendering tab contents
3. And handling routes.

how can we make it to smaller components?